### PR TITLE
Initial file for Bogdan bio

### PR DESCRIPTION
**Pull request by bmatiu**
The more complex the change the more comment. Talking about my bio is a simple thing.

### CODE REVIEW

File: bogdan-bio.myown [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## A little more about only me
+
+*I forgot how to live and where to live
```
Comment (bmatiu) [inline bogdan-bio.myown:3]: Hey this is so true.

Comment (szoellner) [inline bogdan-bio.myown:3]: Me too, almost every day

Comment (DiegoALeon) [inline bogdan-bio.myown:3]: Are you understanding this?

**Comment by bmatiu** [comment]
:+1: this is really good

**Pull request merged**